apps/cmp.c

```diff
@@ -368,7 +368,7 @@ OPTIONS cmp_options[] = {
     {OPT_MORE_STR, 0, 0,
      "negative certificate responses (ip/cp/kup), and revocation responses (rp)"},
     {OPT_MORE_STR, 0, 0,
-     "WARNING: This setting leads to behaviour allowing violation of RFC 4210"},
+     "WARNING: This setting leads to behavior allowing violation of RFC 4210"},
```

Comment:
I thought that traditionally OpenSSL was less US-centric than other software might have been :-D

Reply:
Yeah, I think so too, but nowadays, even for OpenSSL, IT tends to be rather US-centric.